library/system.lisp

```diff
@@ -86,13 +86,14 @@ While the result will always contain microseconds, some implementations may retu
     (lisp UFix ()
       (cl:get-internal-real-time)))
 
-  #+sbcl
-  (declare get-bytes-consed (Unit -> UFix))
-  #+sbcl
+  (declare get-bytes-consed (Unit -> (Optional UFix)))
```

Comment:
Should be `Integer` since SBCL may return that.

Reply:
SBCL says it might return a [consed bignum](https://github.com/sbcl/sbcl/blob/a1cecc85c4bd176c2ff3838ccc1e4c510156cbef/src/code/gc.lisp#L60C23-L60C36), but I'm garbage collecting beforehand so it should avoid numbers that big. Changing it to Integer though